ghost/core/test/integration/services/email-service/email-event-storage.test.js

```diff
@@ -462,7 +462,7 @@ describe('EmailEventStorage', function () {
         // Message and code not changed
         assert.equal(permanentFailures.models[0].get('message'), 'Not delivering to previously bounced address');
         assert.equal(permanentFailures.models[0].get('code'), 605);
-        assert.equal(permanentFailures.models[0].get('enhanded_code'), null);
+        assert.equal(permanentFailures.models[0].get('enhanded_code'), undefined);
```

Comment:
Should this actually be `enhanced_code` anyway?

Reply:
Yes, updated